include/depthimage_to_laserscan/DepthImageToLaserScan.h

```diff
@@ -217,9 +273,13 @@ namespace depthimage_to_laserscan
     float range_max_; ///< Stores the current maximum range to use.
     int scan_height_; ///< Number of pixel rows to use when producing a laserscan from an area.
     std::string output_frame_id_; ///< Output frame_id for each laserscan.  This is likely NOT the camera's frame_id.
+    tf::TransformListener listener_; ///< TF listener for retrieving transform between frames.
+    float height_min_;
+    float height_max_;
+    static const int OFFSET_BOTTOM_ROW = 150;
```

Comment:
Assuming that the TF is correct (should be correct relative to the map so min height should be from the map floor).